pkgs/top-level/python-packages.nix

```diff
@@ -565,6 +565,8 @@ in {
 
   apcaccess = callPackage ../development/python-modules/apcaccess { };
 
+  APC-Temp-fetch = callPackage ../development/python-modules/APC-Temp-fetch { };
```

Comment:
Please normalize the attribute name. According to the [python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names)